src/mc-fle2-find-equality-payload-private-v2.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef MC_FLE2_FIND_EQUALITY_PAYLOAD_PRIVATE_H
+#define MC_FLE2_FIND_EQUALITY_PAYLOAD_PRIVATE_H
```

Comment:
```suggestion
#ifndef MC_FLE2_FIND_EQUALITY_PAYLOAD_PRIVATE_V2_H
#define MC_FLE2_FIND_EQUALITY_PAYLOAD_PRIVATE_V2_H
```

To distinguish between mc-fle2-find-equality-payload-private.h